<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareSamplesDialog</class>
 <widget class="QDialog" name="CompareSamplesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compare Samples</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>p. value</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="compareButton">
     <property name="text">
      <string>Compare Sets</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="minFoldDiff">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
     <property name="value">
      <double>-100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Min LOG2 Fold Diff</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Min. Intensity</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QDoubleSpinBox" name="minIntensity">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QDoubleSpinBox" name="minPValue">
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QComboBox" name="correctionBox">
     <item>
      <property name="text">
       <string>No Correction</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bonferroni</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Holms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Benjamini</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Set Missing Values</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="missingValue">
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>FDR Correction</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Min. Good Samples</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QSpinBox" name="minGoodSamples">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="7">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Set 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QListWidget" name="filelist1">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DropOnly</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Set 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="fileCount1">
        <property name="text">
         <string>0 samples selected</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="fileCount2">
        <property name="text">
         <string>0 samples selected</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QListWidget" name="filelist2">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DropOnly</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CompareSamplesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
